### DISCUSSION TITLE
Fun Fact: It's the second anniversary of Scratch Addons today!

### DISCUSSION THREAD

**Discussion by TheColaber** [ScratchAddons/ScratchAddons#5128] [Discussion]
Yay! 🥳 🥳 🎉 🎉

**Comment by rgantzos** [comment]
Happy birthday =D

**Comment by BroJac5246** [comment]
What should we do to celebrate? 🎉

**Reply by rgantzos** [reply]
The server is holding a party right now.

**Reply by BroJac5246** [reply]
Hmm... too bad I don't use Discord.

**Reply by Secret-chest** [reply]
Why are they called servers? They are all on Discord and they don't serve anything.

**Reply by BroJac5246** [reply]
> Why are they called servers? They are all on Discord and they don't serve anything.

Because they 'serve' messages?

**Reply by rgantzos** [reply]
> Why are they called servers? They are all on Discord and they don't serve anything.

Technically they’re called guilds, but the community calls them servers.

**Reply by BroJac5246** [reply]
> > Why are they called servers? They are all on Discord and they don't serve anything.
> 
> Technically they’re called guilds, but the community calls them servers.

Interesting.

**Reply by CST1229** [reply]
> Technically they’re called guilds, but the community calls them servers.

I think they were used to be called guilds (and still are in stuff like the API), but they have been since renamed to servers.

**Reply by rimopa** [reply]
> > Technically they’re called guilds, but the community calls them servers.
> 
> I think they were used to be called guilds (and still are in stuff like the API), but they have been since renamed to servers.

yes
![image](https://user-images.githubusercontent.com/97112832/210633702-30d705c2-dcc6-451a-8535-39ce886b9984.png)

**Reply by Secret-chest** [reply]
> Why are they called servers? They are all on Discord and they don't serve anything.

Offtopic, but now that I got this in my email, it feels a bit... cringy? I'm old-school and server means some kind of email server, web server, Minecraft server, even IRC server, but in this case, Discord is the server.
